metadata-ingestion/src/datahub/ingestion/source/s3/config.py

```diff
@@ -20,18 +20,14 @@
 
 
 class DataLakeSourceConfig(PlatformSourceConfigBase, EnvBasedSourceConfigBase):
-    path_specs: Optional[List[PathSpec]] = Field(
+    path_specs: List[PathSpec] = Field(
         description="List of PathSpec. See below the details about PathSpec"
```

Comment:
Can we replace the `See below` part of the description with a link#anchor to the PathSpec documentation?